generators/app/notebookConverter.js

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+
+exports.processNotebookFolder = (folderPath, generator) => {
+    let errors = [];
+    let count = findNotebookFiles(folderPath, errors, generator);
+
+    if (count <= 0) {
+        generator.log("No valid notebooks found in " + folderPath + (errors.length > 0 ? '.\n' + errors.join('\n'): ''));
```

Comment:
end of line can be different in different platforms

Reply:
[https://stackoverflow.com/questions/43771470/os-eol-in-typescriptnode-js](https://stackoverflow.com/questions/43771470/os-eol-in-typescriptnode-js)

---
In reply to: [464649192](https://github.com/llali/generator-azuredatastudio/pull/23#discussion_r464649192) [](ancestors = 464649192)